packages/nimble-components/src/anchor-tabs/tests/anchor-tabs.spec.ts

```diff
@@ -255,6 +255,11 @@ describe('AnchorTabs', () => {
             const specType = getSpecTypeByNamedList(test, focused, disabled);
             // eslint-disable-next-line @typescript-eslint/no-loop-func
             specType(test.name, async () => {
+                // Can't focus a link without an href, so set them all here.
+                tab(0).href = 'foo';
```

Comment:
Any reason to not set the `href` attribute in the fixture template? I don't see any tests that rely on that attribute not being set. Should there be?

Reply:
There are tests that will click on a tab (e.g. to change the active tab), and if an href is set it will attempt a navigation, which aborts the test run. I don't really need to (and can't practically) test navigating, so I generally want no hrefs set.

Reply:
Let's go ahead and make a separate fixture relevant for these tests then